test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
+    {
+        ProjectTestExtension _webProject;
+        ProjectItemTestExtension _libManConfig;
+        const string _projectName = @"TestProjectCore20";
+        const string _libman = "libman.json";
+        private string _initialLibmanFileContent;
+        private string _pathToLibmanFile;
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            _webProject = Solution[_projectName];
+            _libManConfig = _webProject[_libman];
+            _pathToLibmanFile = Path.Combine(SolutionRootPath, _projectName, _libman);
+            _initialLibmanFileContent = File.ReadAllText(_pathToLibmanFile);
+
+            if (File.Exists(_libManConfig.FullPath))
+            {
+                string projectPath = Path.Combine(SolutionRootPath, _projectName);
+                _libManConfig.Delete();
+                Helpers.FileIO.WaitForDeletedFile(projectPath, Path.Combine(projectPath, _libman), caseInsensitive: false);
```

Comment:
>Path.Combine(projectPath, _libman) [](start = 63, length = 34)

_libManConfig.FullPath?

Reply:
Updated in iteration 2.

---
In reply to: [205527861](https://github.com/aspnet/LibraryManager/pull/332#discussion_r205527861) [](ancestors = 205527861)